org/teams.yaml

```diff
@@ -190,6 +190,20 @@ teams:
           - kyessenov
           - mandarjog
           - nrjpoddar
+      WG - Product Security Maintainers:
+        description: Maintainers for the Product Security working group.
```

Comment:
@howardjohn are we excluding all TOC/Steering members from the list?

I think we should have members of TOC (Josh/me) & Steering (Kevin) here as long as we continue to participate as per the WG charter.

Reply:
I wasn't sure if you 3 were members or just "TOC that attends meetings". Happy to add all 3 here.

Reply:
TOC automatically becoming members has not worked out for us so I would suggest that we start with the 3 contributing members. Thanks!

Reply:
done